lib/views/variant-view.dart

```diff
@@ -232,17 +243,32 @@ class _SortablePageState extends State<SortablePage> {
           onPressed: () {
             setState(() {
               user["pinned"] = !user["pinned"];
-              if (user["pinned"] && !user["generated"]) {
-                selections.add(user["accession"]);
-              } else {
-                selections.remove(user["accession"]);
-              }
             });
           },
         ),
       )
     ));
 
+    lister.add(DataCell(
+        Align(
+          alignment: Alignment.centerRight,
+          child: IconButton(
+            icon: user["selected"] ? const Icon(Icons.done_sharp): const Icon(Icons.do_disturb),
```

Comment:
May be better to have like an empty box when an item is not selected - the do_disturb icon (believe the circle with a line through it?) may give off the impression it's not selectable

Reply:
Looks good otherwise